sw/airborne/modules/computer_vision/opticflow/size_divergence.c

```diff
@@ -114,8 +114,9 @@ float get_size_divergence(struct flow_t *vectors, int count, int n_samples)
     }
   }
 
-  // calculate the mean divergence:
-  float mean_divergence = mean_f(divs, used_samples);
+  float mean_divergence;
+  // calculate the mean divergence or use 0 when used_samples is still 0
+  mean_divergence = mean_f(divs, used_samples);
```

Comment:
why this change?

Reply:
It looks like that is the remainder from when I fixed it in size_div. Instead of unfixing it I changed it again. Next commit should restore it to the original state.